test/basic/extractor.cpp

```diff
@@ -30,6 +30,7 @@ BOOST_AUTO_TEST_CASE(annotator_test_basic)
 
     Database db(true);
     Extractor extractor(buffer.c_str(), buffer.size(), "xml", db);
+    db.build_rtree();
```

Comment:
@allierowan Why this change?  The `Database db(true);` line above should signal the `Extractor` to build the RTree - if it's not, then this test is possibly failing legitimately.